frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -55,6 +55,14 @@ function ReaderBookmark:addToMainMenu(menu_items)
             self:onShowBookmark()
         end,
     }
+    if not Device:isTouchDevice() then
+        menu_items.add_bookmarks = {
+            text = _("Toggle bookmarks"),
```

Comment:
I guess it should be singular `Toggle bookmark` (or `Toggle page bookmark` ?)
And for consistency, name the menu item index similar to its text, eg: `menu_items.toggle_bookmark` 

You could even add a function (untested):
```lua
function ReaderBookmark:isCurrentPageBookmarked()
    local pn_or_xp
    if self.ui.document.info.has_pages then
        pn_or_xp = self.view.state.page
    else
        pn_or_xp = self.ui.document:getXPointer()
    end
    return self:getDogearBookmarkIndex(pn_or_xp) and true or false
end
```
and use it as `text_func = function() return self:isCurrentPageBookmarked() and "Remove bookmark for current page" or "Bookmark current page"`

Reply:
Yes, that's better.